docs/contributing/database-migrations/edd.mdx

```diff
@@ -190,6 +190,13 @@ Create a PR moving the future scripts.
 - `DbScripts_future` to `DbScripts`, prefix the script with the current date, but retain the
   existing date.
 - `dbo_future` to `dbo`.
+  <bitwarden>
+    <li>
+      Create a ticket in Jira with a `Due Date` of the release date to ensure future migrations are
+      merged in and ready to be executed. Set the ticket which created the future migration as a
```

Comment:
⛏️ ticket "that".

Reply:
As a native speaker, that truly is a ⛏️  😂  but I don't disagree

Reply:
blame the 🤖